src/geometric_algebra/nursery/talk.lean

```diff
@@ -86,49 +94,54 @@ variables (K : Type u) [field K]
 
 variables (V : Type u) [add_comm_group V] [vector_space K V]
 
-structure GA
-(G : Type u)
-[ring G] extends algebra K G :=
```

Comment:
Apparently `extends` is a terrible idea if the target has fewer arguments than the currently declared type